dist/form-for.js

```diff
@@ -170,6 +171,18 @@ var formFor;
             this.requiredLabel = value;
         };
         /**
+        * Disable using Bootstrap Glyphicons
+        */
+        FormForConfiguration.prototype.disableGlyphicons = function () {
```

Comment:
What if, instead of a boolean setting for glyph icons, we exposed a string setting that allowed users to specify any help icon they want? (This way users of other font-icon sets would be able to specify alternatives as well.)